client/src/stores/SimulationStore.ts

```diff
@@ -38,12 +38,18 @@ const useSimulationStore = create<SimulationState>((set) => ({
     verticalAngle: 0.5236, // 30 degrees
   },
   actions: {
-    start: () =>
+    start: (launchConditions?: Partial<LaunchConditions>) =>
```

Comment:
I added launch conditions as an argument to the start function. I'm not married to this at all, I just felt like I was asking for a race condition if I called `updateLaunchConditions` and then `start` soon after.

Reply:
That sounds reasonable to me. I think I was initially picturing that changes to the launch condition inputs would be applied to the launch conditions state immediately, like on every change event, so that when you clicked "start" the launch conditions were already applied, but this makes just as much sense